libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -362,6 +400,11 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     this.changeActiveView(index, 'months');
   }
 
+  public changeYear(year: number) {
+    const activeYear = parseInt(this.activeYear);
+    this.changeActiveView(year - activeYear, 'years');
```

Comment:
Consider simplifying as `this.activeYear` is  a string and intended use as presentation in the markup. It also seems a bit redundant to convert it to a string and the parsing it back into a number :)
Alternatively change `this.activeYear` to a number instead...
```suggestion
    this.changeActiveView(year - this.activeMonth.year, 'years');
```

Reply:
Changing its type is going to require calling .toString() in two other places: private getHelperOptions() and get navigatedYear(), so I've gone with your suggestion as it is much cleaner 👍